pyincore/utils/huapdoutputprocess.py

```diff
@@ -0,0 +1,409 @@
+# Copyright (c) 2021 University of Illinois and others. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+
+import json
+import pandas as pd
+
+
+class HUADislOutputProcess:
+    """This class converts csv results outputs of Population dislocation analysis to json format.
+
+    Args:
+        pd_count (obj): IN-CORE dataset for Joplin Population Dislocation (PD) results.
+        pd_count_path (obj): A fallback for the case that Joplin PD object is not provided.
+            For example a user wants to directly pass in csv files, a path to PD results.
+
+    """
+    HUPD_CATEGORIES = ["housing_unit_characteristics",
+                       "housing_unit_dislocations",
+                       "housing_unit_in_total",
+                       "population_dislocations",
+                       "population_in_total"
+                       ]
+
+    def __init__(self, pd_count, pd_count_path=None):
+        if pd_count_path:
+            self.pd_count = pd.read_csv(pd_count_path, low_memory=False)
+        else:
+            self.pd_count = pd_count_path.get_dataframe_from_csv(low_memory=False)
```

Comment:
this should be `self.pd_count = pd_count.get_dataframe_from_csv(low_memory=False)`

Reply:
Oops. Done